scripts/ci/update_downstream.sh

```diff
@@ -35,10 +35,10 @@ git add pom.xml
 
 if [[ "${COMMIT_MESSAGE}" =~ \[force[^\]]*\] ]]; then
   FORCE_TOKEN=$(echo "${COMMIT_MESSAGE}" | sed "s|^.*\(\[force[^]]*\]\).*$|\1|g")
-  git commit --allow-empty -m "${FORCE_TOKEN} Update upstream version to ${VERSION}"
+  git commit --allow-empty -m "${FORCE_TOKEN} Update upstream community-repo version to ${VERSION}"
```

Comment:
```suggestion
  git commit --allow-empty -m "${FORCE_TOKEN} Update community-repo version to ${VERSION}"
```
IMO the upstream bump was clear enough as is, since there is only community-repo upstream.
If we really want to make it clearer by specifying that we're bumping the community-repo version though, then I wouldn't mention the "upstream" at all anymore. This is the only upstream project we have anyways.